### PR TITLE
Barry's feedback

### PR DESCRIPTION
**Pull request by richsalz**

### CODE REVIEW

File: draft-rsalz-2028bis.md [hunk 1]
```diff
@@ -223,8 +224,9 @@ technical, architectural, procedural, and policy matters
 pertaining to the Internet and its enabling technologies.
 
 The members of the IAB are nominated by NomCom,
-and are approved by the ISOC board.  The IETF Chair is an ex-officio member
-of the board.
+and are confirmed by the Internet Society Board.
+The IETF Chair and the IRTF Chair are an ex-officio members
+of the IAB.
```
Comment (larseggert) [inline draft-rsalz-2028bis.md:229]: The IETF Chair is **not** an ex officio member of the IAB - they are a regular member.

File: draft-rsalz-2028bis.md [hunk 2]
```diff
@@ -260,12 +262,13 @@ through the IETF using the rules in this document.
...
 
-Details of the organization and operation of the IRTF, the ISRG,
+Details of the organization and operation of the IRTF, the IRSG,
 and its Research Groups may be found in {{IRTF}}.
 
 ## The IETF Trust
 
-The IETF Trust holds the copyrights for IETF documents including RFCs, and
+The IETF Trust holds IETF trademarks,
+the copyrights for IETF documents including RFCs, and
```
Comment (larseggert) [inline draft-rsalz-2028bis.md:271]: The Trust also holds other assets, such as for IANA.

File: draft-rsalz-2028bis.md [hunk 3]
```diff
@@ -294,7 +297,7 @@ The IETF Executive Director handles the daily tasks and management,
 and is overseen by the LLC Board of Directors.
 
 {{ISOCIETF, Section 6}} describes the legal relationship between the
-LLC and ISOC.
+LLC and Internet Society.
```
Comment (larseggert) [inline draft-rsalz-2028bis.md:300]:
```suggestion
LLC and the Internet Society.
```

**Review by larseggert — CHANGES_REQUESTED** [review]

File: draft-rsalz-2028bis.md (hunk 1 above)
Comment (richsalz) [reply draft-rsalz-2028bis.md]: Fixed, thanks.

File: draft-rsalz-2028bis.md (hunk 2 above)
Comment (richsalz) [reply draft-rsalz-2028bis.md]: Fixed, thanks.  "Among our weaponry are ... "

**Pull request merged**